app/time_tracking/event/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Event(models.Model):
+    title       = models.CharField(max_length=120)
+    description = models.TextField(blank=True, null=True)
```

Comment:
if you have optional text field, it must be `blank=True, default=''` and not `blank=True, null=True` as `null` is not `''` and in that case you allow to store two different values with the same meaning which could create bugs in the code